configs/letsencrypt/init-letsencrypt.sh

```diff
@@ -39,10 +42,8 @@ if ! [ -x "$(command -v docker-compose)" ]; then
 fi
 
 if [ -d "$data_path" ]; then
-  read -p "Existing data found for $domains. Continue and replace existing certificate? (y/N) " decision
-  if [ "$decision" != "Y" ] && [ "$decision" != "y" ]; then
-    exit
-  fi
+  echo "Existing data found for $domains at $data_path."
+  exit 0
```

Comment:
This is still causing me issues in a fresh install.
* First it says `Existing data found`
* I delete this block, then it complains:
   ```
   ERROR: for redis  Cannot create container for service redis: Conflict. The container name "/redis" is already in use by container "fb7fae07185909d2540bb9bcf4c811827f1b92a7e8f33f7ea5c22693136197ea". You have to remove (or rename) that container to be able to reuse that name.
  ```

This is when using `eb create production --single` followed by `eb deploy production`

Reply:
This error is unrelated, what's happened is most likely EB didn't properly kill all containers with the failed deployment.  I think in this case you need to go to `/var/app/current` and execute `sudo docker-compose down -v` then redeploy. 

I've seen this before elsewhere, since it's brought up, here again, looks like we're gonna need to add our own solution to it.

Reply:
It happens even if I `eb terminate` and start over

Reply:
i'll retry today then, this was working normally for me yesterday